.github/workflows/e2e-node.yml

```diff
@@ -0,0 +1,46 @@
+name: End-to-end node tests
+
+on:
+  push:
+
+env:
+  CI: true
+jobs:
+  e2e-node:
+    # Dependabot cannot access secrets, so it doesn't have a token to authenticate to ESS.
+    # Since all the other jobs of this workflow depend on this one, skipping it should
+    # skip the entire workflow.
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    runs-on: ${{ matrix.os }}
+    environment:
+      name: ${{ matrix.environment-name }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [16.x]
+        # NSS does not support static client registration, which we rely on for testing.
+        environment-name: ["ESS Dev-Next"]
```

Comment:
What about "ESS Production"?

Reply:
Production still running on 1.1, access grants are not implemented yet (that I'm aware of).